go.mod

```diff
@@ -1,236 +1,248 @@
 module github.com/rancher/kube-api-auth
 
-go 1.17
+go 1.19
+
+// note: replace once 1.1.1 has been cut
+replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230705223603-201b4da5bdaf
```

Comment:
Isn't v1.1.1 already cut https://github.com/rancher/wrangler/releases/tag/v1.1.1?

Reply:
The comment is in error - the replaced commit is for changes after 1.1.1. We need to replace once 1.1.2 (or whatever the next release of wrangler is) has been cut.